src/organisms/index.ts

```diff
@@ -0,0 +1,19 @@
+export { DataGrid } from './DataGrid/DataGrid';
+export type { EdsDataGridProps as DataGridProps } from '@equinor/eds-data-grid-react';
```

Comment:
Was not sure if this should be moved down (to the other EDS reexport) or not, since it is part of "our" components since we redesign it, but then again these props are just a straight reexport 🤔

Reply:
Yeah wasn't too sure either, felt like placing them next to eachother like this so it is obvious

Reply:
Yeah, i think its fine, just wanted to point it out incase it was just overlooked c: